src/clj/puppetlabs/services/legacy_routes/legacy_routes_service.clj

```diff
@@ -27,7 +28,9 @@
           master-route-config (master-core/get-master-route-config
                                 master-ns
                                 config)
-          master-route-handler (-> (master-core/root-routes handle-request)
+          jruby-service (tk-services/get-service this :JRubyPuppetService)
+          master-route-handler (-> (master-core/root-routes handle-request
+                                                            jruby-service)
```

Comment:
I guess there have been recent attempts at doing something like this in other PRS, like in #887, where a dummy function was passed in, `(constantly nil)`, instead of the real reference.  Not totally sure how I feel about that, I guess.  Might work for now but if there ever ended up being a reason for the "real" depedency to have to be used in a legacy routes context, I'd be worried that this would be broken without us realizing it.